### ISSUE TITLE
Add controls for site copyright and licensing

### ISSUE THREAD

**Issue by brechtcs** [beakerbrowser/beaker#1204]
As discussed in https://github.com/beakerbrowser/beaker/issues/1187#issuecomment-428013143, it might be legally necessary for to give site owners some control over the "forkability" of their site. Opening this issue to gather some ideas about how to implement this.

**Comment by Butterfly** [comment]
I was sitting here thinking I couldn't remember any precedence for this... and then I remembered [Creative Commons](https://creativecommons.org/).

Is that anything that might be able to be worked into what you all have in mind? It's something that's at least somewhat familiar to folks these days, primarily k/t image sharing.. :smile:

**Comment by RangerMauve** [comment]
Yeah, something in the `dat.json` that cued clients like Beaker or the Dat CLI that forking is not allowed would be awesome.

It wouldn't stop people from copying the content, but it'd at least give them a warning that the author doesn't want people to copy their content.

**Comment by brechtcs** [comment]
Right, I think adding a simple boolean field `allow_forks` to the dat manifest file would do the trick. As mentioned in the other thread this would give us three cases to handle:

1. `allow_forks === true`: In this case we can safely show the 'Make editable copy' option in the UI.
2. `allow_forks === false`: This is also pretty straightforward; just hide the option in the UI.
3. `allow_forks === undefined`: This is more tricky. I'd say show the option in the UI, but trigger a user warning before proceeding when it's clicked. I'd also include `null` and all other non-valid boolean values in this scenario, because falling back on JS truthy/falsy behaviour feels a bit suspect in matters of consent.

Also it's probably not a bad idea to take into account any licenses already added as `links` to the manifest. For example by requiring an "Terms and conditions" modal to accept these licences when forking. That might tie into issue https://github.com/beakerbrowser/beaker/issues/1097 as well.
